qiskit/tools/visualization/_bloch.py

```diff
@@ -39,6 +39,7 @@
 #    OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ###############################################################################
 
+# pylint: disable=ungrouped-imports
```

Comment:
Is it possible to reorder the imports a bit to avoid doing this pylint ignore? In general it's better not to overuse them (which we already do).
Non-blocking though.

Reply:
Oops this was leftover from https://github.com/Qiskit/qiskit-terra/pull/911/commits/8f0e17445ecb78f32120d1b72f9d492a5e1a1836 which definitely needed this. But I fixed the pylint errors in a second commit so when I reverted this it stayed around. I'll revert that other commit and it should correct this.